docs/guides/dynamo_run.md

```diff
@@ -521,6 +521,7 @@ More fully-featured Backend engines (used by `dynamo-run`):
 
 The input defaults to `in=text`. The output will default to `out=mistralrs` engine, unless it is disabled with `--no-default-features` in which case vllm is used.
 
+<a id="extra_engine_arguments"></a>
```

Comment:
You don't need this, Markdown generates them. It's linked from the top:
https://github.com/ai-dynamo/dynamo/blob/main/docs/guides/dynamo_run.md#extra-engine-arguments